src/sdk/resources/personalDrive.ts

```diff
@@ -1,6 +1,7 @@
 import { IHttpClient } from '../http';
 import { PagedResource } from './resource';
 import { PersonalDriveMatchesResource } from './personalDriveMatches';
+import { Account } from '..';
```

Comment:
Why was this needed?

Reply:
`Account` was part of global API until recent TypeScript update..